src/esp/physics/PhysicsManager.cpp

```diff
@@ -448,5 +448,11 @@ void PhysicsManager::setObjectBBDraw(int physObjectID,
         0, *existingObjects_[physObjectID]->BBNode_, drawables);
   }
 }
+
+const scene::SceneNode& PhysicsManager::getObjectSceneNode(int physObjectID) {
+  assertIDValidity(physObjectID);
```

Comment:
I will recommend:
-) CORRADE_ASSERT instead of CHECK within assertIDValidity;
-) std::unordered_map instead of std::map, which is faster, for existingObjects_;

But certainly none of this needs to be changed in this PR. Just FYI.

Reply:
Thanks for the suggestions. Should do a refactor pass over some of that in the near future.